### PR TITLE
Update ktlint to 0.39

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/414730916066338/1199145345336726/f
Tech Design URL: 
CC: 

**Description**:
We updated kotlin to 1.4 version however ktlint is still in 0.36, which does not support the new features in kotlin 1.4. This means that if we try to use any of the new features in 1.4, ktlint will complain.

(Support for 1.4 was added in ktlint in 0.38)

**Steps to test this PR**:
1. fresh install the app
2. check core funcionality works

1. update an already install app
2. check core functionality works 

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)

### CODE REVIEW

**Comment by aitorvs** [comment]
FYI: The changes in the kt files are caused by the update of ktlint itself

File: app/src/main/java/com/duckduckgo/app/notification/NotificationRegistrar.kt [hunk 1]
```diff
@@ -71,11 +71,13 @@ class NotificationRegistrar @Inject constructor(
...
-        ChannelType.FILE_DOWNLOADED,
-        ChannelType.TUTORIALS
-    ))
+    private val channels = (
+        listOf(
+            ChannelType.FILE_DOWNLOADING,
+            ChannelType.FILE_DOWNLOADED,
+            ChannelType.TUTORIALS
+        )
+        )
```
Comment (marcosholgado) [inline app/src/main/java/com/duckduckgo/app/notification/NotificationRegistrar.kt:80]: This is so odd 🤷‍♂️

File: styleguide/STYLEGUIDE.md [hunk 2]
```diff
@@ -3,23 +3,18 @@
...
+###  Code formatting
 
-We've provided a coding style that can be imported into Android Studio.
-The file can be found [here](ddg-settings.zip).
+To adhere to codestyle, please run `./gradlew spotlessApply` to autoformat in order to fix any CI issues.
 
-* From Android Studio click on File -> Import Settings
+If you want to do this automatically upon commit we recommend the following `pre-commit` hook.
 
-![Setting kotlin style screenshot](import_settings_one.png)
```
Comment (marcosholgado) [inline styleguide/STYLEGUIDE.md:13]: Can you also delete the DDG.xml along with all the screenshots and the ddg-settings.zip file from the styleguide folder?

**Review by marcosholgado — CHANGES_REQUESTED** [review]
Thanks @aitorvs, just needs deleting the old screenshots, xml and zip file :)

File: app/src/main/java/com/duckduckgo/app/notification/NotificationRegistrar.kt (hunk 1 above)
Comment (aitorvs) [reply app/src/main/java/com/duckduckgo/app/notification/NotificationRegistrar.kt:80]: interestingly enough, DDG.xml formats it the same way, just adding more continuation indent 🤷

Comment (aitorvs) [reply app/src/main/java/com/duckduckgo/app/notification/NotificationRegistrar.kt]: As it seems there's an extra parenthesis, I removed it. That seems what was causing this weird formatting in both ktlint and ddg.xml. See https://github.com/duckduckgo/Android/pull/1006/commits/2f32cdca635b20a82f5dfa818bd0491bb3cfda35

File: styleguide/STYLEGUIDE.md (hunk 2 above)
Comment (aitorvs) [reply styleguide/STYLEGUIDE.md:13]: Done

**Review by marcosholgado — APPROVED** [review]
LGTM! Nice one @aitorvs 👍

**Pull request merged**